api/src/services/tus/data-store.ts

```diff
@@ -39,7 +37,6 @@ export class TusDataStore extends DataStore {
 		super();
 
 		if (config.constants.CHUNK_SIZE !== null) this.chunkSize = config.constants.CHUNK_SIZE;
-		if (config.constants.MAX_SIZE !== null) this.maxSize = config.constants.MAX_SIZE;
```

Comment:
Any reason behind this removal?

Reply:
Upon investigation this looks to be left over dead-code and can be removed. maxSize is handled at the server level.